### ISSUE TITLE
[Froala 3] Fullscreen with adjusted toolbarButtons throws Uncaught TypeError: Cannot read property 'getBoundingClientRect' of undefined

### ISSUE THREAD

**Issue by Naartti** [froala/wysiwyg-editor#3551]
##### Expected behavior.
Clicking the fullscreen toolbar button should expand Froala without throwing an error.

##### Actual behavior.
Froala expands into fullscreen but displays the error *Uncaught TypeError: Cannot read property 'getBoundingClientRect' of undefined*.

<img width="619" alt="image" src="https://user-images.githubusercontent.com/14360953/61199980-3b57b680-a6e0-11e9-843b-4682f984a3ab.png">

##### Steps to reproduce the problem.
**Step 1**
Set option toolbarButton to the example toolbarButton config found at [https://www.froala.com/wysiwyg-editor/examples/toolbar-buttons-3](url)
```js
toolbarButtons = {
    moreText: {
      buttons: ['bold', 'italic', 'underline', 'strikeThrough', 'subscript', 'superscript', 'fontFamily', 'fontSize', 'textColor', 'backgroundColor', 'inlineClass', 'inlineStyle', 'clearFormatting'],
      align: 'left',
      buttonsVisible: 3
    },


    moreParagraph: {
      buttons: ['alignLeft', 'alignCenter', 'formatOLSimple', 'alignRight', 'alignJustify', 'formatOL', 'formatUL', 'paragraphFormat', 'paragraphStyle', 'lineHeight', 'outdent', 'indent', 'quote'],
      align: 'left',
      buttonsVisible: 3
    },

    moreRich: {
      buttons: ['insertLink', 'insertImage', 'insertVideo', 'insertTable', 'emoticons', 'fontAwesome', 'specialCharacters', 'embedly', 'insertFile', 'insertHR'],
      align: 'left',
      buttonsVisible: 3
    },

    moreMisc: {
      buttons: ['undo', 'redo', 'fullscreen', 'print', 'getPDF', 'spellChecker', 'selectAll', 'html', 'help'],
      align: 'right',
      buttonsVisible: 2
    }
  }
```

**Step 2**
Remove any of the groups, e.g. ```moreRich```.

Or, change toolbarButtons to an array also throws the error.
```js
toolbarButtons = ['undo', 'redo', 'fullscreen']
```

**Step 3**
Click the fullscreen-button.

##### Editor version.
3.0.2

##### OS.
Mac OS Mojave (v. 10.14.5)

##### Browser.
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Safari Version 12.1.1

**Comment by uwe-pichl** [comment]
You can find a duplicate of this issue in the angular-part of the Froala:
https://github.com/froala/angular-froala-wysiwyg/issues/320

**Comment by inssein** [comment]
Duplicate of https://github.com/froala/wysiwyg-editor/issues/3483 as well.

**Comment by casualuser** [comment]
fixed in 3.0.4
